tests/foreman/ui/test_reporttemplates.py

```diff
@@ -567,9 +567,10 @@ def test_positive_gen_entitlements_reports_multiple_formats(
         assert DEFAULT_SUBSCRIPTION_NAME in tree_result
 
 
+@pytest.mark.rhel_ver_list([7, 8, 9])
 @pytest.mark.tier3
 def test_positive_generate_all_installed_packages_report(
-    session, setup_content, rhel7_contenthost, target_sat
+    session, setup_content, rhel_contenthost, target_sat
```

Comment:
There are a few more tests in these modules that are using rhel7_contenthost, do you think those can be parametrized as well?

Reply:
They might be able to be parameterized but that is not in scope with this PR